pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml

```diff
@@ -736,6 +737,7 @@ broken-packages:
   - changelog-d # failure in job https://hydra.nixos.org/build/253689337 at 2024-03-31
   - changelog-d # failure in job https://hydra.nixos.org/build/255671571 at 2024-04-16
   - changelog-d # failure in job https://hydra.nixos.org/build/257082502 at 2024-04-27
+  - changelog-d # failure in job https://hydra.nixos.org/build/259607598 at 2024-05-15
```

Comment:
🤨

Reply:
Oh, it's defined in `non-hackage-packages.nix`, so doesn't actually get marked broken.